meta-dts-distro/recipes-extended/minio-cli/minio-cli_git.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Simple and fasst CLI tool to manage MinIO deployments."
```

Comment:
@DaniilKl Do we plan to upstream this recipe?

Reply:
@macpijan, I tried to upstream it ( [check patches here](https://lists.openembedded.org/g/openembedded-devel/search?p=recentpostdate%2Fsticky%2C%2Cgspread%2C20%2C2%2C0%2C0&q=minio)) but then found out that there is already a recipy for this tool in `meta-openembedded` (added in commit `89653bab7189bfe82419ebaf12145a6f1616e67e`):

```bash
λ f 'minio'
./meta-oe/recipes-extended/minio
./meta-oe/recipes-extended/minio/minio
```

And now I am a bit confused about how have I missed this one and why `meta-networking` maintainer did not mentioned about this. So, I don't know for now.

Reply:
@DaniilKl Have you responded in that thread that the recipe is already there?

We use too old version of Yocto to use this recipe and have to have it here for now?

Reply:
@macpijan, no, I have not informed the `meta-networking` maintainer, going to do it in a few days.

About the `minio` recipy from `meta-openembedded`. The recipy does bould but has some runtime issues:

![image](https://github.com/user-attachments/assets/97f79f98-b422-4a33-a659-ea9a8ebbfdfc)

And I cannot say what the cause is, because it is a GO tool and it is known that it is a bit difficult to build such tools under `bitbake`. So, I would stay with our recipy and maybe signal the issue in mailing lists or push our recipy to `meta-networking`.

Reply:
Actually I think the `minio` recipy misses some `glibc-utils` dependencies, because `getent` is a part of it. But I am not sure that the recipy will bring a working `mc` CLI even after adding `getent`, because when I had this dependency missing - I had slightly different error message, but it may be `mc` CLI revision differences. :thinking: 

I suppose this discussion should be continued in Yocto mailing lists with the maintainers of the appropriate layers.